### PR TITLE
Bump ModelingToolkit v9 Catalyst v14

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
@isaacsas do you think you can find the time to update the jump problems in here? They use the mutating form and probably don't need too much, but until this bump occurs most things won't test Catalyst v14

**Comment by isaacsas** [comment]
Ok

**Comment by isaacsas** [comment]
I’ll try to update tomorrow.

File: lib/JumpProblemLibrary/Project.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ RuntimeGeneratedFunctions = "7e49a35a-f44a-4d26-94aa-eba1b4ca6b47"
 
 [compat]
 Aqua = "0.5"
-Catalyst = "13"
+Catalyst = "14"
```
Comment (ChrisRackauckas) [inline lib/JumpProblemLibrary/Project.toml:12]:
```suggestion
Catalyst = "13"
```

**Comment by ChrisRackauckas** [comment]
I rolled back Jump in here to get the SDE one released.

**Pull request merged**